### PR TITLE
Add maintainer zhongnan

### PR DESCRIPTION
**Pull request by CCongWang**
### Description

We have followed internal maintainer nomination process, and add maintainer: https://github.com/zhongnansu

### Issues Resolved

N/A

### Check List

- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by peterzhuamazon — APPROVED** [review]
Please also add @zhongnansu into the CODEOWNER file.

Thanks.

**Comment by CCongWang** [comment]
Thanks @peterzhuamazon ! just added

**Review by peterzhuamazon — APPROVED** [review]

**Comment by peterzhuamazon** [comment]
Need one more approval right here.

**Review by kavilla — APPROVED** [review]

**Comment by zhongnansu** [comment]
@kavilla @CCongWang Thanks team!

**Pull request merged**
